matrix-sdk/src/main/java/org/matrix/androidsdk/data/MyUser.java

```diff
@@ -124,17 +124,20 @@ public void onSuccess(Void info) {
      * Build the lists of identifiers
      */
     private void buildIdentifiersLists() {
-        List<ThirdPartyIdentifier> identifiers = mDataHandler.getStore().thirdPartyIdentifiers();
         mEmailIdentifiers = new ArrayList<>();
         mPhoneNumberIdentifiers = new ArrayList<>();
-        for (ThirdPartyIdentifier identifier : identifiers) {
-            switch (identifier.medium) {
-                case ThreePid.MEDIUM_EMAIL:
-                    mEmailIdentifiers.add(identifier);
-                    break;
-                case ThreePid.MEDIUM_MSISDN:
-                    mPhoneNumberIdentifiers.add(identifier);
-                    break;
+        List<ThirdPartyIdentifier> identifiers = mDataHandler.getStore().thirdPartyIdentifiers();
+        //NPE reported on playstore
+        if (mDataHandler.getStore() != null) {
```

Comment:
this is used just above :) the `if`should be before line 129

Reply:
oups, done thx